src/sqlfluff/cli/commands.py

```diff
@@ -558,7 +568,16 @@ def lint(
                         "annotation_level": annotation_level,
                     }
                 )
-        click.echo(json.dumps(github_result))
+        file_output = json.dumps(github_result)
+
+    if file_output:
```

Comment:
Nitpick: Did you consider creating a reusable function for this section of code?

Reply:
Good idea :+1